docs/UserGuide.md

```diff
@@ -84,11 +85,11 @@ Here are some symbols used throughout this user guide:
       1. [Listing all patients: `list`](#listing-all-patients-list)
       2. [Finding patients: `find`](#finding-patients-find)
       3. [Viewing all details of a patient: `focus` `-p`](#viewing-all-details-of-a-patient-focus--p)
-      4. [Listing all patients for today: `view` `--today`](#listing-all-patients-for-today-view---today)
-   9. [Viewing tasks](#listing-all-tasks-view--p---all)
-      1. [Listing all tasks: `view` `-p` `-all`](#listing-all-tasks-view--p---all)
-      2. [Viewing all tasks of a patient: `view` `-p`](#viewing-all-tasks-of-a-patient-view--p)
+   9. [Viewing tasks](#listing-all-tasks-of-a-patient-view--p)
+      1. [Listing all tasks: `view` `-p` `-all`](#listing-all-tasks-of-a-patient-view--p)
+      2. [Listing all tasks of a patient: `view` `-p`](#listing-all-tasks-of-all-patients-view--p---all)
       3. [Listing all tasks for a particular day: `view`](#listing-all-tasks-for-a-particular-day-view)
+      4. [Listing all patients for today: `view` `--today`](#listing-all-patients-for-today-view---today)
```

Comment:
Since this changes the order of features, perhaps it may be good to update the order of commands in the command summary table as well.